error: Failed to create folder in jenkins: 500 Server Error

When running "jx install cluster --provider=oce", the staging environment project is created successfully in github but failed to create pipeline in jenkins server.  The error is "Created environment staging
error: Failed to create the hhuiliu folder in jenkins: 500 Server Error
" . Could you please give me a hand on this? Btw, I can access the jenkins server successfully and create folder in the jenkins server.
`
Creating default staging and production environments
Using git provider GitHub at https://github.com


About to create repository environment-loonroot-staging on server https://github.com with user hhuiliu


Creating repository hhuiliu/environment-loonroot-staging
Creating git repository hhuiliu/environment-loonroot-staging
Cloning into '.'...
remote: Counting objects: 77, done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 77 (delta 3), reused 0 (delta 0), pack-reused 67
Unpacking objects: 100% (77/77), done.
[master 0840ee7] Use correct namespace for environment
 2 files changed, 2 insertions(+), 2 deletions(-)
[master c90ff4d] Add environment configuration
 1 file changed, 9 insertions(+)
Counting objects: 85, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (82/82), done.
Writing objects: 100% (85/85), 15.70 KiB | 0 bytes/s, done.
Total 85 (delta 47), reused 0 (delta 0)
remote: Resolving deltas: 100% (47/47), done.
To https://hhuiliu:a32af9afa5a38a70d4c01d5479d866ba97574e5f@github.com/hhuiliu/environment-loonroot-staging.git
 * [new branch]      master -> master
Branch master set up to track remote branch master from origin.
Pushed git repository to https://github.com/hhuiliu/environment-loonroot-staging

Created environment staging
error: Failed to create the hhuiliu folder in jenkins: 500 Server Error

`

Could you please give me a hand on that? Thanks a lot.

And the jenkins server log has no error about this.

The jenkins pod log is like below:
`[opc@demotest .jx]$ kubectl logs pod/jenkins-bb88ccdb4-2bcbp
Running from: /usr/share/jenkins/jenkins.war
webroot: EnvVars.masterEnvVars.get("JENKINS_HOME")
Jul 02, 2018 7:18:58 AM org.eclipse.jetty.util.log.Log initialized
INFO: Logging initialized @1962ms to org.eclipse.jetty.util.log.JavaUtilLog
Jul 02, 2018 7:18:58 AM winstone.Logger logInternal
INFO: Beginning extraction from war file
Jul 02, 2018 7:19:00 AM org.eclipse.jetty.server.handler.ContextHandler setContextPath
WARNING: Empty contextPath
Jul 02, 2018 7:19:00 AM org.eclipse.jetty.server.Server doStart
INFO: jetty-9.4.z-SNAPSHOT, build timestamp: 2017-11-21T21:27:37Z, git hash: 82b8fb23f757335bb3329d540ce37a2a2615f0a8
Jul 02, 2018 7:19:13 AM org.eclipse.jetty.webapp.StandardDescriptorProcessor visitServlet
INFO: NO JSP Support for /, did not find org.eclipse.jetty.jsp.JettyJspServlet
Jul 02, 2018 7:19:13 AM org.eclipse.jetty.server.session.DefaultSessionIdManager doStart
INFO: DefaultSessionIdManager workerName=node0
Jul 02, 2018 7:19:13 AM org.eclipse.jetty.server.session.DefaultSessionIdManager doStart
INFO: No SessionScavenger set, using defaults
Jul 02, 2018 7:19:13 AM org.eclipse.jetty.server.session.HouseKeeper startScavenging
INFO: Scavenging every 660000ms
Jenkins home directory: /var/jenkins_home found at: EnvVars.masterEnvVars.get("JENKINS_HOME")
Jul 02, 2018 7:19:14 AM org.eclipse.jetty.server.handler.ContextHandler doStart
INFO: Started w.@114a85c2{/,file:///var/jenkins_home/war/,AVAILABLE}{/var/jenkins_home/war}
Jul 02, 2018 7:19:14 AM org.eclipse.jetty.server.AbstractConnector doStart
INFO: Started ServerConnector@21ec5d87{HTTP/1.1,[http/1.1]}{0.0.0.0:8080}
Jul 02, 2018 7:19:14 AM org.eclipse.jetty.server.Server doStart
INFO: Started @18351ms
Jul 02, 2018 7:19:14 AM winstone.Logger logInternal
INFO: Winstone Servlet Engine v4.0 running: controlPort=disabled
Jul 02, 2018 7:19:16 AM jenkins.InitReactorRunner$1 onAttained
INFO: Started initialization
Jul 02, 2018 7:19:20 AM hudson.ClassicPluginStrategy createClassJarFromWebInfClasses
WARNING: Created /var/jenkins_home/plugins/job-dsl/WEB-INF/lib/classes.jar; update plugin to a version created with a newer harness
Jul 02, 2018 7:19:23 AM jenkins.InitReactorRunner$1 onAttained
INFO: Listed all plugins
Jul 02, 2018 7:19:32 AM jenkins.InitReactorRunner$1 onAttained
INFO: Prepared all plugins
Jul 02, 2018 7:19:32 AM jenkins.InitReactorRunner$1 onAttained
INFO: Started all plugins
Jul 02, 2018 7:19:34 AM okhttp3.internal.platform.Platform log
INFO: ALPN callback dropped: HTTP/2 is disabled. Is alpn-boot on the boot class path?
Jul 02, 2018 7:19:35 AM okhttp3.internal.platform.Platform log
INFO: ALPN callback dropped: HTTP/2 is disabled. Is alpn-boot on the boot class path?
Jul 02, 2018 7:19:35 AM jenkins.InitReactorRunner$1 onAttained
INFO: Augmented all extensions
Jul 02, 2018 7:19:36 AM jenkins.model.Jenkins setBuildsAndWorkspacesDir
INFO: Using non default workspaces directories: ${JENKINS_HOME}/workspace/${ITEM_FULLNAME}.
Jul 02, 2018 7:19:36 AM org.jenkinsci.plugins.jx.resources.GlobalPluginConfiguration$1 doRun
INFO: Waiting for Jenkins to be started
Jul 02, 2018 7:19:36 AM jenkins.InitReactorRunner$1 onAttained
INFO: Loaded all jobs
Jul 02, 2018 7:19:36 AM jenkins.util.groovy.GroovyHookScript execute
INFO: Executing /var/jenkins_home/init.groovy.d/init-docker-registry-env.groovy
Jul 02, 2018 7:19:36 AM hudson.model.AsyncPeriodicWork$1 run
INFO: Started Download metadata
Jul 02, 2018 7:19:36 AM hudson.model.AllView migrateLegacyPrimaryAllViewLocalizedName
INFO: JENKINS-38606 detected for AllView in hudson.model.Hudson@eb699b9; renaming view from All to all
Jul 02, 2018 7:19:37 AM org.springframework.context.support.AbstractApplicationContext prepareRefresh
INFO: Refreshing org.springframework.web.context.support.StaticWebApplicationContext@78503dd: display name [Root WebApplicationContext]; startup date [Mon Jul 02 07:19:37 UTC 2018]; root of context hierarchy
Jul 02, 2018 7:19:37 AM org.springframework.context.support.AbstractApplicationContext obtainFreshBeanFactory
INFO: Bean factory for application context [org.springframework.web.context.support.StaticWebApplicationContext@78503dd]: org.springframework.beans.factory.support.DefaultListableBeanFactory@74c0e519
Jul 02, 2018 7:19:37 AM org.springframework.beans.factory.support.DefaultListableBeanFactory preInstantiateSingletons
INFO: Pre-instantiating singletons in org.springframework.beans.factory.support.DefaultListableBeanFactory@74c0e519: defining beans [filter,legacy]; root of factory hierarchy
Adding docker registry DOCKER_REGISTRY = 10.96.207.19:5000
Jul 02, 2018 7:19:37 AM jenkins.util.groovy.GroovyHookScript execute
INFO: Executing /var/jenkins_home/init.groovy.d/tcp-slave-agent-port.groovy
Jul 02, 2018 7:19:38 AM jenkins.InitReactorRunner$1 onAttained
INFO: Completed initialization
Jul 02, 2018 7:19:38 AM hudson.WebAppMain$3 run
INFO: Jenkins is fully up and running
Jul 02, 2018 7:19:41 AM hudson.model.UpdateSite updateData
INFO: Obtained the latest update center data file for UpdateSource default
Jul 02, 2018 7:19:43 AM hudson.model.DownloadService$Downloadable load
INFO: Obtained the updated data file for hudson.tasks.Maven.MavenInstaller
Jul 02, 2018 7:19:43 AM hudson.model.AsyncPeriodicWork$1 run
INFO: Finished Download metadata. 6,641 ms
--> setting agent port for jnlp
--> setting agent port for jnlp... done
Jul 02, 2018 7:36:15 AM com.squareup.okhttp.internal.Platform$JdkWithJettyBootPlatform getSelectedProtocol
INFO: ALPN callback dropped: SPDY and HTTP/2 are disabled. Is alpn-boot on the boot class path?
Jul 02, 2018 7:36:16 AM org.jenkinsci.plugins.github.webhook.subscriber.PingGHEventSubscriber onEvent
INFO: PING webhook received from repo <https://github.com/hhuiliu/environment-jx-dummy>!
Jul 02, 2018 7:36:17 AM jenkins.branch.MultiBranchProject$BranchIndexing run
INFO: hhuiliu/environment-jx-dummy #20180702.073615 branch indexing action completed: SUCCESS in 1.6 sec
Jul 02, 2018 7:36:17 AM org.jenkinsci.plugins.jx.resources.BuildSyncRunListener onStarted
INFO: starting polling build job/hhuiliu/job/environment-jx-dummy/job/master/1/
Jul 02, 2018 7:36:18 AM com.squareup.okhttp.internal.Platform$JdkWithJettyBootPlatform getSelectedProtocol
INFO: ALPN callback dropped: SPDY and HTTP/2 are disabled. Is alpn-boot on the boot class path?
Jul 02, 2018 7:36:18 AM io.fabric8.kubernetes.client.internal.VersionUsageUtils alert
WARNING: The client is using resource type 'customresourcedefinitions' with unstable version 'v1beta1'
Jul 02, 2018 7:36:18 AM okhttp3.internal.platform.Platform log
INFO: ALPN callback dropped: HTTP/2 is disabled. Is alpn-boot on the boot class path?
Jul 02, 2018 7:36:19 AM org.jenkinsci.plugins.jx.resources.BuildSyncRunListener upsertBuild
INFO: Created  pipeline activity hhuiliu-environment-jx-dummy-master-1
Jul 02, 2018 7:36:26 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 1
Jul 02, 2018 7:36:26 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:36:26 AM hudson.slaves.NodeProvisioner$StandardStrategyImpl apply
INFO: Started provisioning Kubernetes Pod Template from kubernetes with 1 executors. Remaining excess workload: 0
Jul 02, 2018 7:36:36 AM hudson.slaves.NodeProvisioner$2 run
INFO: Kubernetes Pod Template provisioning successfully completed. We have now 2 computer(s)
Jul 02, 2018 7:36:36 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher launch
WARNING: Error in provisioning; agent=KubernetesSlave name: maven-wvsmp, template=PodTemplate{inheritFrom='', name='maven', slaveConnectTimeout=0, label='jenkins-maven', serviceAccount='jenkins', nodeSelector='', volumes=[HostPathVolume [mountPath=/var/run/docker.sock, hostPath=/var/run/docker.sock], org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@5bdd66e6, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ce00d9b, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ba01507], containers=[ContainerTemplate{name='jnlp', image='jenkinsci/jnlp-slave:3.14-1', workingDir='/home/jenkins', command='', args='${computer.jnlpmac} ${computer.name}', resourceRequestCpu='100m', resourceRequestMemory='128Mi', resourceLimitCpu='', resourceLimitMemory=''}, ContainerTemplate{name='maven', image='jenkinsxio/builder-maven:0.0.340', privileged=true, workingDir='/home/jenkins', command='/bin/sh -c', args='cat', ttyEnabled=true, resourceRequestCpu='400m', resourceRequestMemory='512Mi', resourceLimitCpu='1', resourceLimitMemory='1024Mi'}], envVars=[ContainerEnvVar [getValue()=/home/jenkins/.docker/, getKey()=DOCKER_CONFIG], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_AUTHOR_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_AUTHOR_NAME], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_COMMITTER_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_COMMITTER_NAME], ContainerEnvVar [getValue()=http://jenkins:8080, getKey()=JENKINS_URL], ContainerEnvVar [getValue()=/home/jenkins, getKey()=XDG_CONFIG_HOME], ContainerEnvVar [getValue()=-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m, getKey()=_JAVA_OPTIONS]]}
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [create]  for kind: [Pod]  with name: [null]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:341)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:105)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:227)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:756)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:334)
        ... 8 more

Jul 02, 2018 7:36:36 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave _terminate
INFO: Terminating Kubernetes instance for agent maven-wvsmp
Jul 02, 2018 7:36:36 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave _terminate
WARNING: Failed to delete pod for agent jx/maven-wvsmp: Operation: [delete]  for kind: [Pod]  with name: [maven-wvsmp]  in namespace: [jx]  failed.
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [delete]  for kind: [Pod]  with name: [maven-wvsmp]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.deleteThis(BaseOperation.java:669)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.delete(BaseOperation.java:604)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.delete(BaseOperation.java:48)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave._terminate(KubernetesSlave.java:230)
        at hudson.slaves.AbstractCloudSlave.terminate(AbstractCloudSlave.java:67)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:197)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleDelete(OperationSupport.java:209)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.deleteThis(BaseOperation.java:666)
        ... 12 more

ERROR: Failed to delete pod for agent jx/maven-wvsmp: Operation: [delete]  for kind: [Pod]  with name: [maven-wvsmp]  in namespace: [jx]  failed.
Jul 02, 2018 7:36:46 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 1
Jul 02, 2018 7:36:46 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:36:46 AM hudson.slaves.NodeProvisioner$StandardStrategyImpl apply
INFO: Started provisioning Kubernetes Pod Template from kubernetes with 1 executors. Remaining excess workload: 0
Jul 02, 2018 7:36:56 AM hudson.slaves.NodeProvisioner$2 run
INFO: Kubernetes Pod Template provisioning successfully completed. We have now 2 computer(s)
Jul 02, 2018 7:36:56 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 0
Jul 02, 2018 7:36:56 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:36:56 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher launch
WARNING: Error in provisioning; agent=KubernetesSlave name: maven-vbrfc, template=PodTemplate{inheritFrom='', name='maven', slaveConnectTimeout=0, label='jenkins-maven', serviceAccount='jenkins', nodeSelector='', volumes=[HostPathVolume [mountPath=/var/run/docker.sock, hostPath=/var/run/docker.sock], org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@5bdd66e6, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ce00d9b, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ba01507], containers=[ContainerTemplate{name='jnlp', image='jenkinsci/jnlp-slave:3.14-1', workingDir='/home/jenkins', command='', args='${computer.jnlpmac} ${computer.name}', resourceRequestCpu='100m', resourceRequestMemory='128Mi', resourceLimitCpu='', resourceLimitMemory=''}, ContainerTemplate{name='maven', image='jenkinsxio/builder-maven:0.0.340', privileged=true, workingDir='/home/jenkins', command='/bin/sh -c', args='cat', ttyEnabled=true, resourceRequestCpu='400m', resourceRequestMemory='512Mi', resourceLimitCpu='1', resourceLimitMemory='1024Mi'}], envVars=[ContainerEnvVar [getValue()=/home/jenkins/.docker/, getKey()=DOCKER_CONFIG], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_AUTHOR_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_AUTHOR_NAME], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_COMMITTER_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_COMMITTER_NAME], ContainerEnvVar [getValue()=http://jenkins:8080, getKey()=JENKINS_URL], ContainerEnvVar [getValue()=/home/jenkins, getKey()=XDG_CONFIG_HOME], ContainerEnvVar [getValue()=-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m, getKey()=_JAVA_OPTIONS]]}
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [create]  for kind: [Pod]  with name: [null]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:341)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:105)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:227)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:756)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:334)
        ... 8 more

Jul 02, 2018 7:36:56 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave _terminate
INFO: Terminating Kubernetes instance for agent maven-vbrfc
Jul 02, 2018 7:36:56 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave _terminate
WARNING: Failed to delete pod for agent jx/maven-vbrfc: Operation: [delete]  for kind: [Pod]  with name: [maven-vbrfc]  in namespace: [jx]  failed.
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [delete]  for kind: [Pod]  with name: [maven-vbrfc]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.deleteThis(BaseOperation.java:669)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.delete(BaseOperation.java:604)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.delete(BaseOperation.java:48)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave._terminate(KubernetesSlave.java:230)
        at hudson.slaves.AbstractCloudSlave.terminate(AbstractCloudSlave.java:67)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:197)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleDelete(OperationSupport.java:209)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.deleteThis(BaseOperation.java:666)
        ... 12 more

ERROR: Failed to delete pod for agent jx/maven-vbrfc: Operation: [delete]  for kind: [Pod]  with name: [maven-vbrfc]  in namespace: [jx]  failed.
Jul 02, 2018 7:37:06 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 1
Jul 02, 2018 7:37:06 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:37:06 AM hudson.slaves.NodeProvisioner$StandardStrategyImpl apply
INFO: Started provisioning Kubernetes Pod Template from kubernetes with 1 executors. Remaining excess workload: 0
Jul 02, 2018 7:37:16 AM hudson.slaves.NodeProvisioner$2 run
INFO: Kubernetes Pod Template provisioning successfully completed. We have now 2 computer(s)
Jul 02, 2018 7:37:16 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 0
Jul 02, 2018 7:37:16 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:37:16 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher launch
WARNING: Error in provisioning; agent=KubernetesSlave name: maven-6hzjn, template=PodTemplate{inheritFrom='', name='maven', slaveConnectTimeout=0, label='jenkins-maven', serviceAccount='jenkins', nodeSelector='', volumes=[HostPathVolume [mountPath=/var/run/docker.sock, hostPath=/var/run/docker.sock], org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@5bdd66e6, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ce00d9b, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ba01507], containers=[ContainerTemplate{name='jnlp', image='jenkinsci/jnlp-slave:3.14-1', workingDir='/home/jenkins', command='', args='${computer.jnlpmac} ${computer.name}', resourceRequestCpu='100m', resourceRequestMemory='128Mi', resourceLimitCpu='', resourceLimitMemory=''}, ContainerTemplate{name='maven', image='jenkinsxio/builder-maven:0.0.340', privileged=true, workingDir='/home/jenkins', command='/bin/sh -c', args='cat', ttyEnabled=true, resourceRequestCpu='400m', resourceRequestMemory='512Mi', resourceLimitCpu='1', resourceLimitMemory='1024Mi'}], envVars=[ContainerEnvVar [getValue()=/home/jenkins/.docker/, getKey()=DOCKER_CONFIG], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_AUTHOR_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_AUTHOR_NAME], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_COMMITTER_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_COMMITTER_NAME], ContainerEnvVar [getValue()=http://jenkins:8080, getKey()=JENKINS_URL], ContainerEnvVar [getValue()=/home/jenkins, getKey()=XDG_CONFIG_HOME], ContainerEnvVar [getValue()=-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m, getKey()=_JAVA_OPTIONS]]}
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [create]  for kind: [Pod]  with name: [null]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:341)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:105)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:227)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:756)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:334)
        ... 8 more

Jul 02, 2018 7:37:16 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave _terminate
INFO: Terminating Kubernetes instance for agent maven-6hzjn
ERROR: Failed to delete pod for agent jx/maven-6hzjn: Operation: [delete]  for kind: [Pod]  with name: [maven-6hzjn]  in namespace: [jx]  failed.
Jul 02, 2018 7:37:16 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave _terminate
WARNING: Failed to delete pod for agent jx/maven-6hzjn: Operation: [delete]  for kind: [Pod]  with name: [maven-6hzjn]  in namespace: [jx]  failed.
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [delete]  for kind: [Pod]  with name: [maven-6hzjn]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.deleteThis(BaseOperation.java:669)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.delete(BaseOperation.java:604)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.delete(BaseOperation.java:48)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesSlave._terminate(KubernetesSlave.java:230)
        at hudson.slaves.AbstractCloudSlave.terminate(AbstractCloudSlave.java:67)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:197)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleDelete(OperationSupport.java:209)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.deleteThis(BaseOperation.java:666)
        ... 12 more

Jul 02, 2018 7:37:26 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 1
Jul 02, 2018 7:37:26 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:37:26 AM hudson.slaves.NodeProvisioner$StandardStrategyImpl apply
INFO: Started provisioning Kubernetes Pod Template from kubernetes with 1 executors. Remaining excess workload: 0
Jul 02, 2018 7:37:36 AM hudson.slaves.NodeProvisioner$2 run
INFO: Kubernetes Pod Template provisioning successfully completed. We have now 2 computer(s)
Jul 02, 2018 7:37:36 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Excess workload after pending Kubernetes agents: 0
Jul 02, 2018 7:37:36 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud provision
INFO: Template: Kubernetes Pod Template
Jul 02, 2018 7:37:36 AM org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher launch
WARNING: Error in provisioning; agent=KubernetesSlave name: maven-9jzp4, template=PodTemplate{inheritFrom='', name='maven', slaveConnectTimeout=0, label='jenkins-maven', serviceAccount='jenkins', nodeSelector='', volumes=[HostPathVolume [mountPath=/var/run/docker.sock, hostPath=/var/run/docker.sock], org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@5bdd66e6, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ce00d9b, org.csanchez.jenkins.plugins.kubernetes.volumes.SecretVolume@4ba01507], containers=[ContainerTemplate{name='jnlp', image='jenkinsci/jnlp-slave:3.14-1', workingDir='/home/jenkins', command='', args='${computer.jnlpmac} ${computer.name}', resourceRequestCpu='100m', resourceRequestMemory='128Mi', resourceLimitCpu='', resourceLimitMemory=''}, ContainerTemplate{name='maven', image='jenkinsxio/builder-maven:0.0.340', privileged=true, workingDir='/home/jenkins', command='/bin/sh -c', args='cat', ttyEnabled=true, resourceRequestCpu='400m', resourceRequestMemory='512Mi', resourceLimitCpu='1', resourceLimitMemory='1024Mi'}], envVars=[ContainerEnvVar [getValue()=/home/jenkins/.docker/, getKey()=DOCKER_CONFIG], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_AUTHOR_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_AUTHOR_NAME], ContainerEnvVar [getValue()=jenkins-x@googlegroups.com, getKey()=GIT_COMMITTER_EMAIL], ContainerEnvVar [getValue()=jenkins-x-bot, getKey()=GIT_COMMITTER_NAME], ContainerEnvVar [getValue()=http://jenkins:8080, getKey()=JENKINS_URL], ContainerEnvVar [getValue()=/home/jenkins, getKey()=XDG_CONFIG_HOME], ContainerEnvVar [getValue()=-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Dsun.zip.disableMemoryMapping=true -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m, getKey()=_JAVA_OPTIONS]]}
io.fabric8.kubernetes.client.KubernetesClientException: Operation: [create]  for kind: [Pod]  with name: [null]  in namespace: [jx]  failed.
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:62)
        at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:71)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:341)
        at org.csanchez.jenkins.plugins.kubernetes.KubernetesLauncher.launch(KubernetesLauncher.java:105)
        at hudson.slaves.SlaveComputer$1.call(SlaveComputer.java:291)
        at jenkins.util.ContextResettingExecutorService$2.call(ContextResettingExecutorService.java:46)
        at jenkins.security.ImpersonatingExecutorService$2.call(ImpersonatingExecutorService.java:71)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
    certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
    DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
    subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
        at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:308)
        at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:268)
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
        at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)
        at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)
        at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:54)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at io.fabric8.kubernetes.client.utils.HttpClientUtils$2.intercept(HttpClientUtils.java:93)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
        at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
        at okhttp3.RealCall.execute(RealCall.java:77)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:379)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:344)
        at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:227)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:756)
        at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:334)
        ... 8 more

`
But it seemed the error is related to https://github.com/jenkins-x/jx/issues/603

Looks like cluster certificate issues.

```
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Hostname kubernetes.default not verified:
certificate: sha256/oP/wMyojFDZGASr4oJgBXCbgmfVaak3mKv4HeKLbMvs=
DN: CN=kubernetes.default, OU=ODX, O=Oracle, L=Austin, ST=Texas, C=US
subjectAltNames: [10.96.0.1, 130.35.3.225, 172.16.11.1, 127.0.0.1, czgenrvmu2w.us-phoenix-1.clusters.oci.oraclecloud.com, kubernetes, kubernetes.default.svc, kubernetes.default.svc.cluster, kubernetes.default.svc.clusters.local]
```
Try to `kubectl edit configmap jenkins` search for 
```
<serverUrl>https://kubernetes.default</serverUrl>
```
and replace with
```
<serverUrl>https://kubernetes.default.svc</serverUrl>
```

We can always change the default to be the full service dns name if that works.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -58,7 +58,7 @@ pipeline {
 
                         sh "cp ./build/linux/jx /usr/bin"
 
-                        sh "./build/linux/jx install --namespace ${TEAM} --helm3 --provider=gke -b --headless --default-admin-password $JENKINS_CREDS_PSW --version 0.0.1614"
+                        sh "jx install --namespace ${TEAM} --helm3 --provider=gke -b --headless --default-admin-password $JENKINS_CREDS_PSW"
 
                         // lets test we have the jenkins token setup
                         sh "jx get pipeline"

--- a/pkg/jx/cmd/common_import.go
+++ b/pkg/jx/cmd/common_import.go
@@ -3,6 +3,7 @@ package cmd
 import (
 	"fmt"
 	"net/url"
+	"time"
 
 	"github.com/jenkins-x/jx/pkg/auth"
 	"github.com/jenkins-x/jx/pkg/gits"
@@ -134,59 +135,71 @@ func (o *CommonOptions) ImportProject(gitURL string, dir string, jenkinsfile str
 		}
 	}
 	org := gitInfo.Organisation
-	folder, err := jenk.GetJob(org)
-	if err != nil {
-		// could not find folder so lets try create it
-		jobUrl := util.UrlJoin(jenk.BaseURL(), jenk.GetJobURLPath(org))
-		folderXml := jenkins.CreateFolderXml(jobUrl, org)
-		err = jenk.CreateJobWithXML(folderXml, org)
+	err = o.retry(10, time.Second*10, func() error {
+		folder, err := jenk.GetJob(org)
 		if err != nil {
-			return fmt.Errorf("Failed to create the %s folder in jenkins: %s", org, err)
-		}
-	} else {
-		c := folder.Class
-		if c != "com.cloudbees.hudson.plugins.folder.Folder" {
-			log.Warnf("Warning the folder %s is of class %s", org, c)
-		}
-	}
-	projectXml := jenkins.CreateMultiBranchProjectXml(gitInfo, gitProvider, credentials, branchPattern, jenkinsfile)
-	jobName := gitInfo.Name
-	job, err := jenk.GetJobByPath(org, jobName)
-	if err == nil {
-		if failIfExists {
-			return fmt.Errorf("Job already exists in Jenkins at %s", job.Url)
+			// could not find folder so lets try create it
+			jobUrl := util.UrlJoin(jenk.BaseURL(), jenk.GetJobURLPath(org))
+			folderXml := jenkins.CreateFolderXml(jobUrl, org)
+			err = jenk.CreateJobWithXML(folderXml, org)
+			if err != nil {
+				return fmt.Errorf("Failed to create the %s folder in jenkins: %s", org, err)
+			}
 		} else {
-			log.Warnf("Job already exists in Jenkins at %s\n", job.Url)
+			c := folder.Class
+			if c != "com.cloudbees.hudson.plugins.folder.Folder" {
+				log.Warnf("Warning the folder %s is of class %s", org, c)
+			}
 		}
-	} else {
-		err = jenk.CreateFolderJobWithXML(projectXml, org, jobName)
-		if err != nil {
-			return fmt.Errorf("Failed to create MultiBranchProject job %s in folder %s due to: %s", jobName, org, err)
-		}
-		job, err = jenk.GetJobByPath(org, jobName)
-		if err != nil {
-			return fmt.Errorf("Failed to find the MultiBranchProject job %s in folder %s due to: %s", jobName, org, err)
-		}
-		log.Infof("Created Jenkins Project: %s\n", util.ColorInfo(job.Url))
-		log.Blank()
-		if !isEnvironment {
-			log.Infof("Watch pipeline activity via:    %s\n", util.ColorInfo(fmt.Sprintf("jx get activity -f %s -w", gitInfo.Name)))
-			log.Infof("Browse the pipeline log via:    %s\n", util.ColorInfo(fmt.Sprintf("jx get build logs %s", gitInfo.PipelinePath())))
-			log.Infof("Open the Jenkins console via    %s\n", util.ColorInfo("jx console"))
-			log.Infof("You can list the pipelines via: %s\n", util.ColorInfo("jx get pipelines"))
-			log.Infof("When the pipeline is complete:  %s\n", util.ColorInfo("jx get applications"))
-			log.Blank()
-			log.Infof("For more help on available commands see: %s\n", util.ColorInfo("https://jenkins-x.io/developing/browsing/"))
-			log.Blank()
-		}
-		log.Info(util.ColorStatus("Note that your first pipeline may take a few minutes to start while the necessary docker images get downloaded!\n\n"))
+		return nil
+	})
+	if err != nil {
+		return err
+	}
 
-		params := url.Values{}
-		err = jenk.Build(job, params)
-		if err != nil {
-			return fmt.Errorf("Failed to trigger job %s due to %s", job.Url, err)
-		}
+	err = o.retry(10, time.Second*10, func() error {
+		projectXml := jenkins.CreateMultiBranchProjectXml(gitInfo, gitProvider, credentials, branchPattern, jenkinsfile)
+		jobName := gitInfo.Name
+		job, err := jenk.GetJobByPath(org, jobName)
+		if err == nil {
+			if failIfExists {
+				return fmt.Errorf("Job already exists in Jenkins at %s", job.Url)
+			} else {
+				log.Warnf("Job already exists in Jenkins at %s\n", job.Url)
+			}
+		} else {
+			err = jenk.CreateFolderJobWithXML(projectXml, org, jobName)
+			if err != nil {
+				return fmt.Errorf("Failed to create MultiBranchProject job %s in folder %s due to: %s", jobName, org, err)
+			}
+			job, err = jenk.GetJobByPath(org, jobName)
+			if err != nil {
+				return fmt.Errorf("Failed to find the MultiBranchProject job %s in folder %s due to: %s", jobName, org, err)
+			}
+			log.Infof("Created Jenkins Project: %s\n", util.ColorInfo(job.Url))
+			log.Blank()
+			if !isEnvironment {
+				log.Infof("Watch pipeline activity via:    %s\n", util.ColorInfo(fmt.Sprintf("jx get activity -f %s -w", gitInfo.Name)))
+				log.Infof("Browse the pipeline log via:    %s\n", util.ColorInfo(fmt.Sprintf("jx get build logs %s", gitInfo.PipelinePath())))
+				log.Infof("Open the Jenkins console via    %s\n", util.ColorInfo("jx console"))
+				log.Infof("You can list the pipelines via: %s\n", util.ColorInfo("jx get pipelines"))
+				log.Infof("When the pipeline is complete:  %s\n", util.ColorInfo("jx get applications"))
+				log.Blank()
+				log.Infof("For more help on available commands see: %s\n", util.ColorInfo("https://jenkins-x.io/developing/browsing/"))
+				log.Blank()
+			}
+			log.Info(util.ColorStatus("Note that your first pipeline may take a few minutes to start while the necessary docker images get downloaded!\n\n"))
 
+			params := url.Values{}
+			err = jenk.Build(job, params)
+			if err != nil {
+				return fmt.Errorf("Failed to trigger job %s due to %s", job.Url, err)
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		return err
 	}
 
 	// register the webhook

